MatiasDure/Project_IMTS
# Bee Swimming (Idle)

**Description**

- Bee imitates fish by swimming in random directions, staying within bounds
- Vertical rotation limit to simulate realistic movement

![BeeSwimming](https://github.com/user-attachments/assets/49c5b2a2-bbf6-43e2-87b2-0c683cadcc0a)

**How to test**

- Go to scene BeeSwimming
- Play
- Bee should swim in random directions
- Horizontal Out Of Bounds (Width + Depth): Bee should immediately rotate to the middle and then start getting random directions again
- Vertical Out Of Bounds: Bee should keep moving forward and just pitch up or down accordingly, in order to adjust
